packages/kernel-6.1/1005-Revert-Revert-drm-fb_helper-improve-CONFIG_FB-depend.patch

```diff
@@ -0,0 +1,36 @@
+From 151701bbe89c73340f386cc5363cefc779c4f3f6 Mon Sep 17 00:00:00 2001
+From: Leonard Foerster <foersleo@amazon.com>
+Date: Fri, 11 Aug 2023 06:41:44 +0000
+Subject: [PATCH] Revert "Revert "drm: fb_helper: improve CONFIG_FB
+ dependency""
+
+This reverts commit 9200a3864170e49e8d076870ee18fad6de4fd356.
+
+Amazon Linux has reverted this upstream commit in order to have
+certain DRM options set to allow building nvidia DKMS. I do not agree
+with reverting upstream fixes that make sense to achieve this. Instead
```

Comment:
nit: I would strike the "I do not agree" sentence since it can be (mis-)read as a personal conflict with the AL upstream. I might be reading too much into the first-person POV in a commit message though; it's unusual enough in my experience that it comes across as a strong statement.

The overall logic behind the commit still makes sense - we don't need to revert an upstream patch because of the alternative approach taken in fd73bff24a78, so we can revert the AL revert.

Reply:
It was not my intention to be or sound hostile. But language sometimes may be received differently than intended by the sender. I have softened the language to hopefully reduce any ways this could be misunderstood.